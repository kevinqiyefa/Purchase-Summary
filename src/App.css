.App {
  text-align: center;
  margin: 0 5em;
  position: relative;
  display: flex;
  justify-content: flex-end;
}
