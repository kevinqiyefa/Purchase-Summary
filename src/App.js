import React, { Component } from 'react';

import './App.css';
import SummaryContainer from './containers/SummaryContainer';
import { configureStore } from './store';
import { Provider } from 'react-redux';
import { fetchItemPricingData } from './store/actions/getItemPricingData';

const store = configureStore();
class App extends Component {
  componentDidMount() {
    store.dispatch(fetchItemPricingData());
  }

  render() {
    return (
      <div className="App">
        <Provider store={store}>
          <SummaryContainer />
        </Provider>
      </div>
    );
  }
}

export default App;
