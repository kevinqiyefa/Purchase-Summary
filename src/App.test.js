import React from 'react';
import App from './App';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('<App />', () => {
  let wrapper;
  it('renders', () => {
    // smoke test!!!
    wrapper = shallow(<App />);
    expect(wrapper.hasClass('App')).toEqual(true);
  });

  it('matches snapshot', () => {
    const serialized = toJson(wrapper);
    expect(serialized).toMatchSnapshot();
  });
});
