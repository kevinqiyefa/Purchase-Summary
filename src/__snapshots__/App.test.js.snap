// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> matches snapshot 1`] = `
<div
  className="App"
>
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <Connect(PurchaseSummary) />
  </Provider>
</div>
`;
